package.json

```diff
@@ -23,25 +23,21 @@
     "babel": "~5",
     "babel-core": "~5",
     "babel-loader": "~5",
-    "babel-runtime": "~5",
-    "coveralls": "~2",
```

Comment:
I still have not figured out automated code coverage on Circle. Something todo...